src/xSTIR/pSTIR/STIR.py

```diff
@@ -893,11 +894,11 @@ def read_from_file(self, filename):
         check_status(self.handle)
         self.read_only = True
 
-    def get_cache_path(self): 
-        return "Nikos"
+    def get_cache_path(self):
+        return self.cache_path
 
-    def set_cache_path(self): 
-        print("NikosSEt")
+    def set_cache_path(self, path):
+        self.cache_path = path
```

Comment:
wouldn't this need a call to the C++ function?

Reply:
so far as I can see, `cache_path` is needed by Nikos' objective function, so my guess is that the latter will obtain it from the `LismodeData` object passed either via `acq_data` argument of `make_Poisson` function or via its method `set_listmode_data` or suchlike - @NikEfth am I getting it right?

Reply:
This function should get the cache path, as we set it in the test6. 
It was meant as means to test if  I can correctly set the path from python but didn't finish it. 

If we set the path correctly, we can remove the get function, otherwise, get_cache_path() is the function we should return.